docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -48,9 +49,11 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
 >
 > [Visual Studio 2022 Lifecycle](/lifecycle/products/visual-studio-2022)
 
-> <sup>2</sup> With .NET 6, the.NET 6.0.100 SDK can be used in version 16.11 for **downlevel** targeting. This means that you're not forced to update your SDK and Visual Studio versions simultaneously. However, you won't be able to target .NET 6 because of limitations in 6.0 features and C# 10 features in version 16.11. This compatibility is specifically for targeting 5.0 and below.
+> <sup>3</sup> With .NET 6, the.NET 6.0.100 SDK can be used in version 16.11 for **downlevel** targeting. This means that you're not forced to update your SDK and Visual Studio versions simultaneously. However, you won't be able to target .NET 6 because of limitations in 6.0 features and C# 10 features in version 16.11. This compatibility is specifically for targeting 5.0 and below.
```

Comment:
Note 1 on line 44:
The .NET 5 SDK will be supported in Visual Studio scenarios until December 2022 when 3.1 goes out of support. MSBuild/Visual Studio supported for longer.
-->
The .NET 5 SDK was supported in Visual Studio scenarios until December 2022 when 3.1 went out of support. MSBuild/Visual Studio supported for longer.

Note 2 on line 46:
Visual Studio 17.3 will be in support until 17.4 ships --> do we still need this note?

VS links on lines 48-50:
Should we move these to after the last note?

Reply:
I rewrote note 2 to be more clear and made the other changes.